lib/Doctrine/Common/DataFixtures/ReferenceRepository.php

```diff
@@ -82,7 +81,7 @@ protected function getIdentifier($reference, $uow)
     {
         // In case Reference is not yet managed in UnitOfWork
         if (! $this->hasIdentifier($reference)) {
-            $class = $this->manager->getClassMetadata($this->getRealClass(get_class($reference)));
+            $class = $this->manager->getClassMetadata(get_class($reference));
```

Comment:
No need to use `getRealClass` here since the getClassMetada is already doing the job.